Tests/SwiftyRequestTests/SwiftyRequestTests.swift

```diff
@@ -224,21 +224,48 @@ class SwiftyRequestTests: XCTestCase {
 
     }
 
+    func assertCharsetISO8859(response: HTTPURLResponse?) {
+        guard let text = response?.allHeaderFields["Content-Type"] as? String,
+            let regex = try? NSRegularExpression(pattern: "(?<=charset=).*(?=$|\\s)"),
+            let match = regex.firstMatch(in: text, range: NSRange(text.startIndex..., in: text)),
+            let range = Range(match.range, in: text) else {
+                assertionFailure("Test no longer valid using URL: \(response?.url?.absoluteString ?? ""). The charset field was not provided.")
+                return
+        }
+
+        let str = String(text[range]).trimmingCharacters(in: CharacterSet(charactersIn: "\"").union(.whitespaces))
+        assert(String(str).lowercased() == "iso-8859-1")
```

Comment:
I think this assert will crash the tests if it fails, perhaps it would be better to throw a custom exception which would still differentiate this error from a normal test failure but should also allow the other tests to run.